### PR TITLE
Fix comments for $groups in AuthGroups.php

### PR DESCRIPTION
**Pull request by iamsyh**
The current comment content does not match what the code is doing. Please feel free to clean up or let me know any suggestions.

Thank you!

### CODE REVIEW

File: src/Config/AuthGroups.php [hunk 1]
```diff
@@ -20,10 +20,12 @@ class AuthGroups extends BaseConfig
      * --------------------------------------------------------------------
      * Groups
      * --------------------------------------------------------------------
-     * The available authentication systems, listed
-     * with alias and class name. These can be referenced
-     * by alias in the auth helper:
-     *      auth('api')->attempt($credentials);
+     * The available groups in the system. 
+     * Where the key is the group name, and the values are an 
+     * array of any additional fields you would like for that group.
```
Comment (kenjis) [inline src/Config/AuthGroups.php:25]:
```suggestion
     * array of the group info.
```

**Comment by kenjis** [comment]
Yes, the comment does not match at all. Good catch!

**Comment by kenjis** [comment]
Please run `composer cs-fix` to fix the coding style.

```diff
   1) shield/src/Config/AuthGroups.php (no_trailing_whitespace_in_comment)
      ---------- begin diff ----------
--- /home/runner/work/shield/shield/src/Config/AuthGroups.php
+++ /home/runner/work/shield/shield/src/Config/AuthGroups.php
@@ -20,8 +20,8 @@
      * --------------------------------------------------------------------
      * Groups
      * --------------------------------------------------------------------
-     * The available groups in the system. 
-     * Where the key is the group name, and the values are an 
+     * The available groups in the system.
+     * Where the key is the group name, and the values are an
      * array of the group info.
      * Whatever value you assign as the key, will be used to refer to the group when using functions such as:
      *      $user->addGroup('superadmin')

      ----------- end diff -----------
```

**Comment by datamweb** [comment]
Hi, run the following commands, before running make sure the commit GPG sign settings are set to global.

```console
Composer cs-fix src/Config/AuthGroups.php
git add src/Config/AuthGroups.php
git commit -m 'style: run cs-fix'
git push origin patch-1
```

**Comment by kenjis** [comment]
Please use `git rebase` instead of `git merge` if you can.
See https://github.com/codeigniter4/CodeIgniter4/blob/develop/contributing/workflow.md#updating-your-branch

File: src/Config/AuthGroups.php [hunk 2]
```diff
@@ -20,10 +20,12 @@ class AuthGroups extends BaseConfig
...
-     * The available authentication systems, listed
-     * with alias and class name. These can be referenced
-     * by alias in the auth helper:
-     *      auth('api')->attempt($credentials);
+     * The available groups in the system. 
+     * Where the key is the group name, and the values are an 
+     * array of the group info.
+     * Whatever value you assign as the key, will be used to refer to the group when using functions such as:
+     *      $user->addGroup('superadmin')
+     * ref: https://github.com/codeigniter4/shield/blob/develop/docs/quickstart.md#change-available-groups for more info
```
Comment (paulbalandan) [inline src/Config/AuthGroups.php:28]:
```suggestion
     * An associative array of the available groups in the system, where the keys are
     * the group names and the values are arrays of the group info.
     *
     * Whatever value you assign as the key will be used to refer to the group when using functions such as:
     *      $user->addGroup('superadmin');
     *
     * @var array<string, array<string, string>>
     * @see https://github.com/codeigniter4/shield/blob/develop/docs/quickstart.md#change-available-groups for more info
```

**Comment by lonnieezell** [comment]
@iamsyh are you able to get these changes fixed up? Would love to merge this.

**Comment by iamsyh** [comment]
> @iamsyh are you able to get these changes fixed up? Would love to merge this.

yes i will do it today. apologies for the delay, have been caught up in things

**Comment by MGatner** [comment]
Looks like one more CS Fixer issue. Your branch also has some merges in it - you can start over in a new PR and cherry-pick your changes, or we can just squash this once it is ready.

File: src/Config/AuthGroups.php [hunk 3]
```diff
@@ -20,10 +20,15 @@ class AuthGroups extends BaseConfig
...
-     * by alias in the auth helper:
-     *      auth('api')->attempt($credentials);
+     * An associative array of the available groups in the system, where the keys are
+     * the group names and the values are arrays of the group info.
+     *
+     * Whatever value you assign as the key will be used to refer to the group when using functions such as:
+     *      $user->addGroup('superadmin');
+     *
+     *
+     * @var array<string, array<string, string>>
```
Comment (paulbalandan) [inline src/Config/AuthGroups.php:30]:
```suggestion
     * @var array<string, array<string, string>>
     *
```

**Review by MGatner — APPROVED** [review]

**Pull request merged**